charts/jenkins/values.yaml

```diff
@@ -343,8 +343,14 @@ controller:
 
   ingress:
     enabled: false
+    # For Kubernetes v1.14+, use 'networking.k8s.io/v1beta1'
```

Comment:
to be honest this isn't really true we should probably remove this flag?

the version is hardcoded except for <= 1.13.0 which is long gone
https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/templates/jenkins-controller-ingress.yaml#L3-L9

Reply:
Updated to 
- `For Kubernetes >=v1.14 and <v1.19, use 'networking.k8s.io/v1beta1'`
- `For Kubernetes v1.19+, use 'networking.k8s.io/v1'`

Reply:
I mean that comment is pointless, the code doesn't actually use what you point there, the proper fix would be to remove the version from there, and hardcode the old version in the jenkins-controller-ingress.yaml

Reply:
Looks like the code does use the apiVersion specified in values https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/templates/jenkins-controller-ingress.yaml#L8
Or am I missing something?

Reply:
it only uses it on <= 1.13.0, check the lines above it. it should be inlined there